src/ucp/proto/proto_am.inl

```diff
@@ -24,6 +24,41 @@
 typedef void (*ucp_req_complete_func_t)(ucp_request_t *req, ucs_status_t status);
 
 
+static UCS_F_ALWAYS_INLINE ucs_status_t
+ucp_am_copy_header_to_internal_buf(ucp_request_t *req, ucs_status_t status)
```

Comment:
Looking thru all this PR I see that status can be NO_RESOURCES (zcopy) or OK (bcopy), so it could be 2 different functions which would wrap common one.